src/encoded/tests/data/inserts/file.json

```diff
@@ -4755,5 +4755,20 @@
         "output_type": "functional conservation quantifications",
         "lab": "/labs/richard-myers/",
         "aliases": ["hongkai-ji:human_elements_scored_file"]
+    },
+    {
+        "_test": "regulatory elements prediction model annotation",
+        "status": "in progress",
+        "uuid": "f4d07aba-8a45-49c9-9656-5379e439c16c",
+        "submitted_by": "facilisi.tristique@potenti.vivamus",
+        "md5sum": "8e5883dd35aaa5bce4721a4e5cc471dd",
+        "accession": "ENCFF318REM",
+        "file_format": "h5ad",
```

Comment:
will it work with TXT file? do we need to update dependencies?

Reply:
Adding to the dependencies